LDSContent/ContentController.swift

```
@@ -221,7 +221,12 @@ extension ContentController {
         let directory = mergedURL.deletingLastPathComponent()
         do {
             try FileManager.default.createDirectory(at: directory, withIntermediateDirectories: true, attributes: nil)
-        } catch {}
+        } catch {
+            if !FileManager.default.fileExists(atPath: directory.path) {
+                // The directory doesn't exist so we know the error was real
+                throw error
```

Instead of logging the error here, just throw it so it bubbles up and logs it appropriately to the function calling mergeCatalogs()